patches/tModLoader/Terraria/ModLoader/TileLoader.cs

```diff
@@ -589,22 +589,15 @@ public static void GetItemDrops(int x, int y, Tile tileCache, bool includeLargeO
 	}
 
 	/// <summary>
-	/// Retrieves the item type that would drop from a tile of the specified type and style. This method is only reliable for modded tile types. This method can be used in <see cref="ModTile.GetItemDrops(int, int)"/> for tiles that have custom tile style logic. If the specified style is not found, style 0 will be checked as a fallback.
+	/// Retrieves the item type that would drop from a tile of the specified type and style. This method is only reliable for modded tile types. This method can be used in <see cref="ModTile.GetItemDrops(int, int)"/> for tiles that have custom tile style logic. If the specified style is not found, a fallback item will be returned if one has been registered through <see cref="ModTile.RegisterItemDrop(int, int[])"/> usage.
```

Comment:
Should probably direct people to use `ModTile.GetItemDrops` where possible, since a tile may have custom drop logic